lib/miq_expression/field.rb

```diff
@@ -82,8 +82,20 @@ def contains(other)
     )
   end
 
+  def column_type
+    if custom_attribute_column?
+      CustomAttribute.where(:name => custom_attribute_column_name, :resource_type => model.to_s).first.try(:field_type)
```

Comment:
Some tests in miq_expression_spec are expecting that nil value (https://github.com/ManageIQ/manageiq/blob/master/spec/models/miq_report_spec.rb#L174) I think the nil return is the proper behavior.